src/main.js

```diff
@@ -0,0 +1,93 @@
+import sort from './sort.js';
+
+const itemList = document.querySelector('#list');
+const tasksArray = JSON.parse(localStorage.getItem('todo')) || [];
+export default class Todo {
+  constructor(description) {
+    this.id = tasksArray.length + 1;
+    this.description = description;
+    this.completed = false;
+  }
+
+  static updateIndex = () => {
+    tasksArray.forEach((data, index) => {
+      data.id = index + 1;
+    });
+  };
+
+  static removeTask = (index) => {
+    tasksArray.splice(index - 1, 1);
```

Comment:
You can provide few comments on your code to make it more descriptive, thereby following the JavaScript best Practice.